upload/storage.go

```diff
@@ -0,0 +1,28 @@
+package upload
+
+import (
+	"fmt"
+
+	"gopkg.in/ini.v1"
+)
+
+type (
+	StorageI interface {
+		Upload(source, dest string) error
+	}
+)
+
+func New(section *ini.Section, protocol string) (StorageI, error) {
```

Comment:
this whole file is called "storage.go". Is it only going to support object storage? If so rename it to object_storage.go to make it more clear.

Reply:
this Interface can be any storage backend

Reply:
Ok!